src/bin/cargo.rs

```diff
@@ -186,6 +207,31 @@ fn execute(flags: Flags, config: &Config) -> CliResult<Option<()>> {
     Ok(None)
 }
 
+fn list_of_alias_commands(config: &Config, command: &String) -> Option<Vec<String>> {
```

Comment:
i see. If the alias is an float for example. The code current ignores that. 

Thanks! I will sort this out. I think I want to use a `CargoResult<Option<Vec<_>>>`.